app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -24,7 +25,7 @@ const MapImpl: React.FunctionComponent = () => {
   // Load features on first render
   const featuresQueryResult = useQuery<FeaturesQuery, FeaturesQueryVariables>(
     featuresQueryDocument, 
-    {variables: {limit: 10, offset: 0},
+    {variables: {aLimit: 10, aOffset: 0, aQuery: {type: FeatureType.State}},
```

Comment:
None of this a* variable name stuff, please. You can name the variables limit, offset, and query.